lib/jasperserver-rails/jasperserver-dsl.rb

```diff
@@ -37,11 +37,12 @@ def run_report(filename, &block)
         ).to_s,
         { cookies: @cookie }
       )
+    end
 
-      # Write file
+    def run_report(filename, &block)
```

Comment:
Excellent. This is the reason I reverted ffe38af4771ddefee27cb23062b3388e1a03474e. I wanted a separate method.